code/game/atoms.dm

```diff
@@ -121,6 +121,7 @@
 	/// A luminescence-shifted value of the last color calculated for chatmessage overlays
 	var/chat_color_darkened
 
+	// Use SET_BASE_PIXEL(x, y) to set these in typepath definitions, it'll handle pixel_x and y for you
```

Comment:
should we lint this?

Reply:
can we?

Reply:
we can regex for `\tbase_pixel_x` underneath typepaths, sure

Reply:
that's right yeah. If you want to avoid the pixel offset at map time that can be another define, and you always want to start offset

Reply:
let's do this later